### DISCUSSION TITLE
Authentication with cookie

### DISCUSSION THREAD

**Discussion by PatrickB100** [grafana/grafana-infinity-datasource#308] [General]
Hi All,

I have an API which authenticates in the following way:

1st command: POST - https://api.local/webrequest -> replies with a cookie: jsessionid=xxxxxxxxxxxxx
2nd command: POST - https://api.local/auth -> authenticate with cookie received in first command in header and username/password in body
3th command: post - https://api.local/webrequest -> query data

How do i handle this authentication? 

Thanks in advance,
Patrick

**Comment by yesoreyeram** [comment]
This kind of auth is not supported
